src/app/data-catalogue/components/dataset-detail/dataset-element-table/dataset-element-table.component.html

```diff
@@ -8,10 +8,10 @@
     </tr>
     @for (attribute of attributes; track attribute.name) {
       <tr class="dataset-element-table__table__row">
-        <td class="dataset-element-table__table__row__column">{{ attribute.name }}</td>
-        <td class="dataset-element-table__table__row__column" [innerHTML]="attribute.description | formatLineBreaks"></td>
-        <td class="dataset-element-table__table__row__column">{{ attribute.type }}</td>
-        <td class="dataset-element-table__table__row__column">{{ attribute.unit }}</td>
+        <td class="dataset-element-table__table__row__column">{{ attribute.name ?? '-' }}</td>
+        <td class="dataset-element-table__table__row__column" [innerHTML]="attribute.description ?? '-' | formatLineBreaks"></td>
+        <td class="dataset-element-table__table__row__column">{{ attribute.type ?? '-' }}</td>
+        <td class="dataset-element-table__table__row__column">{{ attribute.unit ?? '-' }}</td>
```

Comment:
I wonder whether we should add a pipe for these, because they exist in lots of places and we could basically have this in a central place (i.e. if it, at some point, should change to eg an empty string).

Reply:
I added a pipe and replaced all the occurences of this